packages/next/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -39,7 +39,7 @@ export default class MiddlewarePlugin {
   }
 
   createAssets(
-    compilation: any,
+    compilation: webpack5.Compilation,
```

Comment:
I think this was recently changed to `any` but I don't remember why

cc @timneutkens @sokra